src/test/java/system_tests/HttpHandler.java

```diff
@@ -0,0 +1,41 @@
+package system_tests;
+
+import okhttp3.*;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.util.HashMap;
+
+public class HttpHandler {
+  public static Response postRequest(String endPoint, HashMap<String, String> bodyParam) {
+    OkHttpClient client = new OkHttpClient().newBuilder().build();
+    MediaType mediaType = MediaType.parse("application/json");
+    RequestBody body = RequestBody.create(mediaType, new JSONObject(bodyParam).toString());
+    Request request =
+        new Request.Builder()
+            .url(endPoint)
+            .method("POST", body)
+            .addHeader("Content-Type", "application/json")
+            .build();
+
+    Response response;
+    try {
+      response = client.newCall(request).execute();
+    } catch (IOException e) {
+      response = null;
+      System.err.println(e);
+    }
+
+    return response;
```

Comment:
In general i think it's better practice to throw an exception here rather than print to err and return null. But since this method is used in the setup method of the test class, that might not be the right move... what does everyone else think?

Reply:
I'm not sure what to do here either?

Reply:
I think we should throw the exception. This method is used in the setup to essentially create what is the fixture for the test. We shouldn't run the tests if the fixture is missing or incomplete.

Reply:
Agreed. Done.